### PR TITLE
VMcollector: update documentation

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- Change regexp from include to exclude
- Tool now cleans up much more VMs than just the `kyma-integration` ones

**Related issue(s)**

#4718

### CODE REVIEW

**Comment by Halamix2** [comment]
/hold
Until #4718 is merged

File: development/tools/cmd/vmscollector/README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 ## Overview
 
-This command finds and removes orphaned virtual machines (VMs) created by the `kyma-integration` job in a Google Cloud Platform (GCP) project.
+This command finds and removes orphaned virtual machines (VMs) created by Prow jobs in a Google Cloud Platform (GCP) project.
 
-The `kyma-integration` job creates a VM instance to install and test Kyma.
+Prow jobs creates a VM instance to install and test Kyma.
```
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:7]:
```suggestion
Prow jobs create a VM instance to install and test Kyma.
```

File: development/tools/cmd/vmscollector/README.md [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
 Usually, the job also cleans up the instance.
 It can happen, however, that the job is terminated before its clean-up finishes.
 This causes a resource leak that generates unwanted costs.
 The garbage collector finds and removes such VM instances.
 
 There are three conditions used to find instances for removal:
-- The instance name pattern that is specific for the `kyma-integration` job
-- The value of a `job-name` label the instance is annotated with
+- The instance name is not caught by the exclud names regexp
+- The value of a `job-name` label the instance is annotated with is not caught by the exclud labels regexp
```
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:15]:
```suggestion
- The value of the `job-name` label the instance is annotated with is not caught by the exclude labels regexp
```

File: development/tools/cmd/vmscollector/README.md [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
+Prow jobs creates a VM instance to install and test Kyma.
 Usually, the job also cleans up the instance.
 It can happen, however, that the job is terminated before its clean-up finishes.
 This causes a resource leak that generates unwanted costs.
 The garbage collector finds and removes such VM instances.
 
 There are three conditions used to find instances for removal:
-- The instance name pattern that is specific for the `kyma-integration` job
-- The value of a `job-name` label the instance is annotated with
+- The instance name is not caught by the exclud names regexp
```
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:14]:
```suggestion
- The instance name is not caught by the exclude names regexp
```

File: development/tools/cmd/vmscollector/README.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
+- The value of a `job-name` label the instance is annotated with is not caught by the exclud labels regexp
 - The instance `creationTimestamp` value that is used to find instance existing at least for a preconfigured number of hours
```
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:16]:
```suggestion
- The instance `creationTimestamp` value that is used to find instance existing at least for a preconfigured number of hours.
```

File: development/tools/cmd/vmscollector/README.md [hunk 5]
```diff
@@ -2,18 +2,18 @@
...
 Usually, the job also cleans up the instance.
 It can happen, however, that the job is terminated before its clean-up finishes.
 This causes a resource leak that generates unwanted costs.
 The garbage collector finds and removes such VM instances.
 
 There are three conditions used to find instances for removal:
-- The instance name pattern that is specific for the `kyma-integration` job
-- The value of a `job-name` label the instance is annotated with
-- The instance `creationTimestamp` value that is used to find instance existing at least for a preconfigured number of hours
+- The instance name is not caught by the exclude names regexp
```
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:14]:
```suggestion
- The instance name is not caught by the exclude names regexp.
```

File: development/tools/cmd/vmscollector/README.md [hunk 6]
```diff
@@ -2,18 +2,18 @@
...
+- The value of the `job-name` label the instance is annotated with is not caught by the exclude labels regexp
```
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:15]:
```suggestion
- The value of the `job-name` label the instance is annotated with is not caught by the exclude labels regexp.
```

File: development/tools/cmd/vmscollector/README.md [hunk 7]
```diff
@@ -2,18 +2,18 @@
...
 Usually, the job also cleans up the instance.
 It can happen, however, that the job is terminated before its clean-up finishes.
 This causes a resource leak that generates unwanted costs.
 The garbage collector finds and removes such VM instances.
 
 There are three conditions used to find instances for removal:
-- The instance name pattern that is specific for the `kyma-integration` job
-- The value of a `job-name` label the instance is annotated with
-- The instance `creationTimestamp` value that is used to find instance existing at least for a preconfigured number of hours
+- The instance name is not caught by the exclude names regexp.
```
Comment (klaudiagrz) [inline development/tools/cmd/vmscollector/README.md:14]:
```suggestion
- The instance name is not caught by the exclude names regex.
```

Comment (klaudiagrz) [reply development/tools/cmd/vmscollector/README.md:14]: According to Google, it's `regex`, not `regexp` 🤔 
Could you verify and change it in the rest of the document if needed?

Comment (Halamix2) [reply development/tools/cmd/vmscollector/README.md]: Done

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**